src/file.js

```diff
@@ -391,6 +393,14 @@ function getStylesheetHrefs(file) {
         return Buffer.from(link.value);
       }
 
+      if (hasMeaningfulMedia(link.$el)) {
+        if (link.value.includes('?')) {
+          link.value += `&media=${encodeURI(link.$el.attr('media'))}`;
```

Comment:
i don't think we should append query params to the stylesheet links. This could lead to errors as we don't always know how these are processed by the server